pulp_2to3_migration/app/plugin/rpm/pulp_2to3_models.py

```diff
@@ -535,7 +546,28 @@ async def create_pulp3_content(self):
         """
         Create a Pulp 3 Distribution content for saving later in a bulk operation.
         """
-        pass
+        namespaces = [".treeinfo", "treeinfo"]
+        for namespace in namespaces:
+            treeinfo = PulpTreeInfo()
+            try:
+                treeinfo.load(f=os.path.join(self.pulp2content.pulp2_storage_path, '.treeinfo'))
+            except FileNotFoundError:
+                continue
+            self.filename = namespace
+            treeinfo_parsed = treeinfo.parsed_sections()
+            treeinfo_serialized = TreeinfoData(treeinfo_parsed).to_dict(filename=namespace)
+            # The repository must already be migrated in order to migrate the DistributionTree
+            pulp2_r_c_unit = RepositoryContentUnit.objects.get(unit_id=self.pulp2content.pulp2_id)
```

Comment:
As a rule of thumb, we are trying not to query MongoDB after pre-migration. You should have all the data needed available in PostgreSQL at this point.
The data you need is pre-migrated into Pulp2RepoContent table/model.

Reply:
Good point. That actually makes this even simpler. Thanks!